packages/lesswrong/components/messaging/NewConversationDialog.tsx

```diff
@@ -0,0 +1,199 @@
+    },
+    "& .ais-SearchBox-submit": {
+      display: "none",
+    },
+    // This is a class generated by React InstantSearch, which we don't have direct control over so
+    // are doing a somewhat hacky thing to style it.
+    "& .ais-SearchBox-input": {
+      height: "100%",
+      width: "100%",
+      paddingRight: 0,
+      verticalAlign: "bottom",
+      borderStyle: "none",
+      boxShadow: "none",
+      backgroundColor: "transparent",
+      fontSize: "inherit",
+      "-webkit-appearance": "none",
+      cursor: "text",
+      ...theme.typography.body2,
+    },
+  },
+  searchHelp: {
+    [theme.breakpoints.down("sm")]: {
+      display: "none",
+    },
+  },
+  infoIcon: {
+    fontSize: 20,
+    fill: theme.palette.grey[800],
+  },
+  usersList: {
+    overflowY: "auto",
+    // Not the actual height, but makes it fill the space when there are no results
+    height: 1000,
+  },
+  hit: {
+    paddingLeft: 20,
+    paddingRight: 20,
+  },
+  closeIcon: {
+    color: theme.palette.grey[600],
+    width: 20,
+    height: 20,
+    cursor: "pointer",
+  }
+});
+
+const NewConversationDialog = ({
+  isModInbox = false,
+  classes,
+  onClose,
+}: {
+  isModInbox?: boolean;
+  classes: ClassesType;
+  onClose: () => void;
+}) => {
+  const { LWDialog, ErrorBoundary, ExpandedUsersConversationSearchHit, ForumIcon, LWTooltip, Typography } = Components;
+  const currentUser = useCurrentUser();
+  const [query, setQuery] = useState<string>("");
+
+  if (!currentUser) return null;
+
+  return (
+    <AnalyticsContext pageSectionContext="newConversationDialog">
+      <LWDialog open={true} onClose={onClose} dialogClasses={{
+        paper: classes.paper,
+      }}>
+        <div className={classes.root}>
+          <div className={classes.titleRow}>
+            <div>New conversation</div>
+            <ForumIcon icon="Close" className={classes.closeIcon} onClick={onClose} />
+          </div>
+          <InstantSearch
+            indexName={getElasticIndexNameWithSorting("Users", "relevance")}
+            searchClient={getSearchClient()}
+            searchState={{ query }}
+            onSearchStateChange={(x) => setQuery(x.query)}
+          >
+            <div className={classes.resultsColumn}>
+              <div className={classes.searchBoxRow}>
+                <div className={classes.searchInputArea}>
+                  <ForumIcon icon="Search" className={classes.searchIcon} />
+                  {/* Ignored because SearchBox is incorrectly annotated as not taking null for its reset prop, when
+                    * null is the only option that actually suppresses the extra X button.
+                  // @ts-ignore */}
+                  <SearchBox defaultRefinement={query} reset={null} focusShortcuts={[]} autoFocus={true} />
+                </div>
+                <LWTooltip title={`"Quotes" and -minus signs are supported.`} className={classes.searchHelp}>
+                  <InfoIcon className={classes.infoIcon} />
```

Comment:
Should this use a `ForumIcon`?

Reply:
I think it should, but we're going to come back and refactor these search boxes anyway so I'm going to leave it like this to keep it the same as the others